src/Components/test/testassets/TestServer/Pages/PrerenderedHost.cshtml

```diff
@@ -19,6 +19,9 @@
     function loadBootScript(event) {
         event.srcElement.disabled = true;
         var scriptElem = document.createElement('script');
+        scriptElem.onload = function () {
+            Blazor.start();
+        };
```

Comment:
I tried a few things to make dynamic loading of this script work for the test and this was the one that worked. LMK if there is a better way.

Reply:
Instead of dynamically inserting the script tag, it now seems we can hard-code the script tag into the document as normal, just don't put the `start` on it. Then `loadBootScript` is replaced just by calling `Blazor.start()`.

This is nice because it shows we have test coverage for this supported usage pattern.

Reply:
solid plan